### PR TITLE
[exo-v2] adds UUID and cleans Additional Installer

### PR DESCRIPTION
**Pull request by Elorfin**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no

- Adds UUIDs on Exercise and Steps and uses it.
- Generate UUIDs from mysql in updater
- Change updater conditions

@pitrackster @stefk

### CODE REVIEW

File: plugin/exo/Entity/Exercise.php [hunk 1]
```diff
@@ -19,6 +19,13 @@ class Exercise extends AbstractResource
     const TYPE_FORMATIVE = '3';
 
     /**
+     * @var string
+     *
+     * @ORM\Column("uuid", type="string", length=36)
+     */
+    protected $uuid;
```
Comment (stefk) [inline plugin/exo/Entity/Exercise.php]: Why not private?

Comment (Elorfin) [inline plugin/exo/Entity/Exercise.php]: Just an habit. I always put my Entities properties in `protected`. I've changed it to set it to `private`.

**Comment by stefk** [comment]
@Elorfin, I just realized unique constraints/indexes are needed on uuid columns.

**Comment by Elorfin** [comment]
@stefk You're right. I will add them in my opened PR for the exo v2.

**Pull request merged**